satellite/peer.go

```diff
@@ -331,7 +331,7 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, config *Config, ve
 			peer.Transport = peer.Transport.WithObservers(peer.Kademlia.RoutingTable)
 		}
 
-		peer.Kademlia.Service, err = kademlia.NewService(peer.Log.Named("kademlia"), peer.Transport, peer.Kademlia.RoutingTable, config)
+		peer.Kademlia.Service, err = kademlia.NewService(peer.Log.Named("kademlia"), peer.Transport, peer.Kademlia.RoutingTable, nil, config)
```

Comment:
Always curious, when satellite is not a Storagenode, then why run kademlia process on satellite and made known to  outside world. This is just a thought .

Reply:
We're going to remove the satellite from kademlia soon